nixos/modules/services/networking/mosh.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg  = config.services.mosh;
+
+in
+{
+  options.services.mosh = {
+      enable = mkEnableOption "mosh";
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [ mosh ];
+    networking.firewall.allowedUDPPortRanges = [ { from = 60000; to = 61000; } ];
```

Comment:
So this port range is what's used by default, I did think about making the port range configurable and it's a little difficult to do.

There's no global configuration file or environment variables we can set to change it. The way the mosh 'server' end works is after a client authenticates via logging in as ssh, the mosh 'client' end starts an instance of the 'server' end. There are two ways to 'configure' a port range - one is using a command line flag when invoking the client, which ends up using the second way of using a command line flag when invoking the server.

The mosh maintainers have previously closed a PR to add support for an environment variable to change the default port range and instead suggested writing a small mosh wrapper script. See https://github.com/mobile-shell/mosh/pull/381

I can look into writing a wrapper script for the server, but I don't know how to do it: can you call `makeWrapper` from a nixos module?